lib/sqlalchemy/sql/selectable.py

```diff
@@ -3595,9 +3622,9 @@ def subquery(self, name: Optional[str] = None) -> Subquery:
 
         """
 
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325